### PR TITLE
BD-2175: Update Item Recommendations

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

Closes #**BD-2175**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

### CODE REVIEW

File: _docs/_hidden/private_betas/recommendations.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ description: "This reference article covers how to create an Item Recommendation
 
 # Item Recommendations
 
-You can use Item Recommendations to calculate the most popular products in a specific [catalog]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/catalogs/). After you create your recommendation, you can use personalization to insert those top products into your messages.
+You can use Item Recommendations to calculate the most popular products in a specific [catalog]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/catalogs/) or create personalized AI recommendations. After you create your recommendation, you can use personalization to insert those products into your messages.
```
Comment (borisrev) [inline _docs/_hidden/private_betas/recommendations.md:10]: I see a slight possibility of confusion. personalized AI is also for a specific catalog.

Comment (bre-fitzgerald) [inline _docs/_hidden/private_betas/recommendations.md:10]:
```suggestion
You can use Item Recommendations to calculate the most popular products or create personalized AI recommendations for a specific [catalog]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/catalogs/). After you create your recommendation, you can use personalization to insert those products into your messages.
```

Comment (bre-fitzgerald) [reply _docs/_hidden/private_betas/recommendations.md]: reworded!

File: _docs/_hidden/private_betas/recommendations.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ description: "This reference article covers how to create an Item Recommendation
 
 # Item Recommendations
 
-You can use Item Recommendations to calculate the most popular products in a specific [catalog]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/catalogs/). After you create your recommendation, you can use personalization to insert those top products into your messages.
+You can use Item Recommendations to calculate the most popular products or create personalized AI recommendations for a specific [catalog]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/catalogs/). After you create your recommendation, you can use personalization to insert those products into your messages.
 
 {% alert important %}
 Item Recommendations is currently in early access. Contact your Braze customer success manager if you're interested in participating in the early access.
```
Comment (borisrev) [inline _docs/_hidden/private_betas/recommendations.md:13]: sorry one other. Most Popular is in early access. Personalized AI will go to beta soon. Unfortunately different :(

**Review by borisrev — CHANGES_REQUESTED** [review]

**Review by borisrev — APPROVED** [review]

**Pull request merged**